flee/flee.py

```diff
@@ -16,11 +15,7 @@
 else:
 
     def check_args_type(func):
-        @wraps(func)
-        def wrapper(*args, **kwargs):
-            return func(*args, **kwargs)
-
-        return wrapper
+        return func
```

Comment:
This function was being double wrapped when it didn't need to be. Every other function uses this wrapper which slowed down every function call.